components/feature/media/src/main/res/drawable/mozac_feature_media_action_pause.xml

```diff
@@ -4,6 +4,6 @@
     android:viewportWidth="24"
     android:viewportHeight="24">
   <path
-      android:fillColor="#FF000000"
+      android:fillColor="#FFFFFFFF"
```

Comment:
nit: I think omitting the alpha works too
```suggestion
      android:fillColor="#FFFFFF"
```

Reply:
Agreed.  I'll change it to omit alpha.  Thanks,